app/views/components/_govspeak-editor.html.erb

```diff
@@ -47,7 +49,9 @@
         margin_bottom: hint ? 5 : 2,
         classes: "js-app-c-govspeak-editor__preview-button",
         data_attributes: preview_button_data_attributes,
-        type: "button"
+        aria_controls: "app-c-govspeak-editor__preview",
+        aria_describedby: id,
```

Comment:
I'm not 100% sure we want this, I understand that this should be an id of an item that describes the purpose of this button (like a span with hint text), but I wouldn't have thought the textarea does that?

Reply:
It might be good to get @dnkrj's  input on this. I asked him about it in a Slack message and he said we could use the aria-describedby to give context to the button as it is read after the aria-label by screenreaders

Reply:
Sounds good - I'm certainly no expert on aria